### DISCUSSION TITLE
DetailsList: Freezing/Sticky columns

### DISCUSSION THREAD

**Discussion by v-vedas** [microsoft/fluentui#28259] [General] [answered]
Do we have the plan to support Locked/Sticky columns/ Rows in DetailsList component? Like the below, we can specify Frozen Rows and Columns.

![image](https://github.com/microsoft/fluentui/assets/59569793/a99f9675-64d4-4463-b5b0-b9b027faead0)

**Comment by ValentinaKozlova** [comment]
@spmonahan and @ThomasMichon , do you have an input on this discussion?

**Reply by spmonahan** [reply, marked as answer]
There are no plans to add this feature to DetailsList. New feature development is happening in Fluent UI React v9 so I recommend making a [feature request](https://github.com/microsoft/fluentui/issues/new?assignees=&labels=Type%3A+Feature%2CNeeds%3A+Triage+%F0%9F%94%8D&projects=&template=feature-request.yml&title=%5BFeature%5D%3A+) to add this to [DataGrid](https://react.fluentui.dev/?path=/docs/components-datagrid--default).

**Comment by ValentinaKozlova** [comment]
Closing this discussion as it's moved to this one: #28288
